test/Microsoft.AspNetCore.Mvc.Razor.Host.Test/RazorEngineTest.cs

```diff
@@ -0,0 +1,219 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Reflection;
+using Microsoft.AspNetCore.Mvc.Razor.Host;
+using Microsoft.AspNetCore.Razor.Evolution;
+using Microsoft.CodeAnalysis;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Mvc.Razor
+{
+    public class RazorEngineTest
+    {
+        private static Assembly _assembly = typeof(RazorEngineTest).GetTypeInfo().Assembly;
+
+        #region Runtime
+        [Fact]
+        public void RazorEngine_Basic_Runtime()
+        {
+            RunRuntimeTest("Basic");
```

Comment:
Eventually it'd be nice to have our code gen tests look like those in Razor (since we changed it)